src/platform/user/authentication/utilities.js

```diff
@@ -15,14 +17,16 @@ export const ssoKeepAliveEndpoint = () => {
 };
 
 function sessionTypeUrl(type = '', version = 'v0', application = null) {
-  const SESSIONS_URI =
+  const base =
     version === 'v1'
       ? `${environment.API_URL}/v1/sessions`
       : `${environment.API_URL}/sessions`;
+  const params = new URLSearchParams();
 
-  return `${SESSIONS_URI}/${type}/new${
-    application ? `?application=${application}` : ''
-  }`;
+  if (application) params.append('application', application);
+  if (getForceAuth() && version === 'v1') params.append('force', 'true');
```

Comment:
@patrickvinograd / @hinzed1127 based on our discussion today should I change this line to be
```
if (version === 'v1) params.append('force', 'true');
```
I think this would allow us to close this https://github.com/department-of-veterans-affairs/va.gov-team/issues/7076